integration-tests/docker/Dockerfile

```diff
@@ -86,4 +86,4 @@ EXPOSE 8300 8301 8302 8303 8304 8305
 EXPOSE 9092 9093
 
 WORKDIR /var/lib/druid
-ENTRYPOINT export HOST_IP="$(resolveip -s $HOSTNAME)" && /tls/generate-server-certs-and-keystores.sh && exec /usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf
+ENTRYPOINT export HOST_IP="$(curl ifconfig.me)" && /tls/generate-server-certs-and-keystores.sh && exec /usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf
```

Comment:
This needs an internet connection, and relies on an external web service that might go away. I think we can do it without needing one. How about:

```
host `hostname` | perl -nle '/has address (.*)/ && print $1'
```

The two most common dns lookup commands AFAIK are `host` and `dig`, so probably either one would be fine.

Reply:
Agree. And i think @leventov already had a fix based on `dig` here: https://github.com/apache/incubator-druid/pull/6702 which also looks good to me.